diff --git a/node_modules/qrcode-vue3/src/core/QRCodeStyling.ts b/node_modules/qrcode-vue3/src/core/QRCodeStyling.ts
index 8de12a6..33109b1 100644
--- a/node_modules/qrcode-vue3/src/core/QRCodeStyling.ts
+++ b/node_modules/qrcode-vue3/src/core/QRCodeStyling.ts
@@ -38,7 +38,11 @@ export default class QRCodeStyling {
       return;
     }
 
-    this._qr = <QRCode>qr.default(this._options.qrOptions.typeNumber, this._options.qrOptions.errorCorrectionLevel);
+    const qrGenerator = (qr as any).default || qr;
+    this._qr = <QRCode>qrGenerator(
+      this._options.qrOptions.typeNumber, 
+      this._options.qrOptions.errorCorrectionLevel
+    );
     this._qr.addData(this._options.data, this._options.qrOptions.mode || getMode(this._options.data));
     this._qr.make();
     this._canvas = new QRCanvas(this._options);
